tests/conftest.py

```diff
@@ -58,11 +58,16 @@ def cleanup():
         Make sure to delete the created account abd buckets
         """
         try:
+            # Delete all the accounts that were created by this fixture
             bucket_manager = BucketManager()
-            for bucket in bucket_manager.list():
-                bucket_manager.delete(bucket, force=True)
-            for acc in acc_manager_instance.list():
+            for acc in acc_manager_instance.accounts_created:
+                # Delete all the buckets that were created by this account
+                acc_id = acc_manager_instance.status(account_name=acc)["_id"]
+                for bucket in bucket_manager.list(use_wide=True):
+                    if bucket["owner_account"] == acc_id:
+                        bucket_manager.delete(bucket["name"], force=True)
                 acc_manager_instance.delete(account_name=acc)
```

Comment:
As this one is part of another for loop, the same apply here.

Reply:
Added in https://github.com/red-hat-storage/noobaa-sa-ci/pull/49/commits/3b60a4ed4f03f61edca0665cdb7bae7f6925cd44 👍